src/components/Orders/OrderItem.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import {
+    Flex,
+    Box,
+    Image,
+    Text,
+    Heading,
+    Button,
+    VStack,
+} from '@chakra-ui/react';
+import { useHistory } from 'react-router';
+import { axiosInstance } from '../../util/config';
+import { API_BASE_URL } from '../../util/config';
+import { colors } from '../../util/constants';
+const OrderItem = ({ productId }) => {
+    const [title, setTitle] = useState('');
+    const [description, setDescription] = useState('');
+    const [images, setImages] = useState('');
+    const getProductInfo = () => {
+        axiosInstance.get(`/products/${productId}`).then((res) => {
+            const { title, description, images } = res.data.product;
+            setTitle(title);
+            setDescription(description);
+            setImages(images[0]);
+        });
+    };
+
+    const history = useHistory();
+
+    useEffect(() => {
+        getProductInfo();
```

Comment:
Since this is the only thing in the `useEffect`, it would have been fine to write it entirely into the `useEffect` instead of creating a `const`. No need for change though.